packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -83,6 +83,22 @@ describe('toggling dropdown', () => {
       expect(comboBox.opened).to.be.false;
     });
 
+    it('should prevent default for the handled toggle-button click', () => {
+      const event = click(comboBox._toggleElement);
+      expect(event.defaultPrevented).to.be.true;
```

Comment:
nit: I guess an integration test with grid/combo-box would be ideal for testing stuff like this. Checking `event.defaultPrevented` isn't necessarily meaningful in itself.

For example, if we later decided to go with the stop propagation approach, these tests would break but the issue this PR fixes would still not reoccur.

Reply:
Thanks, I was thinking about adding integration tests. Let me submit a separate PR for both combo-box and select.
We did not backport `integration` folder to older branches, so adding ITs with a fix would break cherry-picking.